src/vis_cpu/wrapper.py

```diff
@@ -0,0 +1,124 @@
+"""
+Simple wrapper for basic usage of vis_cpu.
+"""
+import numpy as np
+from . import vis_cpu, conversions
+
+
+def simulate_vis(ants, fluxes, ra, dec, freqs, lsts, beams, pixel_beams=False, 
+                 beam_npix=63, polarized=False, precision=1, 
+                 latitude=-30.7215*np.pi/180.):
+    """
+    Simple wrapper around `vis_cpu` for running simulations. This wrapper 
+    handles the necessary coordinate conversions etc.
+    
+    Parameters
+    ----------
+    ants : dict
+        Dictionary of antenna positions. The keys are the antenna names 
+        (integers) and the values are the Cartesian x,y,z positions of the 
+        antennas (in meters) relative to the array center.
+    
+    fluxes : array_like
+        2D array with the flux of each source as a function of frequency, of 
+        shape (NSOURCES, NFREQS).
+    
+    ra, dec : array_like
+        Arrays of source RA and Dec positions in radians. RA goes from [0, 2 pi] 
+        and Dec from [-pi, +pi].
+    
+    freqs : array_like
+        Frequency channels for the simulation, in Hz.
+    
+    lsts : array_like
+        Local sidereal times for the simulation, in radians. Range is [0, 2 pi].
+    
+    beams : list of UVBeam objects
+        Beam objects to use for each antenna.
+    
+    pixel_beams : bool, optional
+        If True, interpolate the beams onto a pixel grid. Otherwise, use the 
+        UVBeam interpolation method directly. Default: False.
+    
+    beam_npix : int, optional
+        If `pixel_beam == True`, sets the pixel grid resolution along each 
+        dimension (corresponds to the `n_pix_lm` parameter of the 
+        `conversions.uvbeam_to_lm` function). Default: 63.
+    
+    polarized : bool, optional
+        If True, use polarized beams and calculate all available linearly-
+        polarized visibilities, e.g. V_nn, V_ne, V_en, V_ee.
+        Default: False (only uses the 'ee' polarization).
+    
+    precision : int, optional
+        Which precision setting to use for vis_cpu. If set to `1`, uses the 
```

Comment:
Would be great to use eg. `:func:`~vis_cpu` in the docstrings so we get links to the function in the sphinx docs.

Reply:
Fixed.